README.md

```diff
@@ -48,11 +48,12 @@ appInsights.loadAppInsights();
 ```ts
 import { ApplicationInsights } from '@microsoft/applicationinsights-web';
 
-var RNPlugin = new ReactNativePlugin();
+var RNPlugin = new ReactNativePlugin({
+    disableDeviceCollection: true
+});
 var appInsights = new ApplicationInsights({
     config: {
         instrumentationKey: 'YOUR_INSTRUMENTATION_KEY_GOES_HERE',
-        disableDeviceCollection: true,
```

Comment:
Actually, looking [at the current code](https://github.com/microsoft/applicationinsights-react-native/blob/11c770202c908bcc0140ad8476e7898e15329d41/applicationinsights-react-native/src/ReactNativeManualDevicePlugin.ts#L64-L66) this is a configuration is only used from the passed in configuration... The `config` that is passed to the constructor is actually never used.....

Which version are you using (older versions may have referenced it).

Based on what I just reviewed of the classes the correct approach is actually
- Remove the "config" from the constructor as it's not used (unless it's referenced in another class -- I just looked at the manual one)
- The config location "works" as currently defined, but would be more correctly located in the `extensionConfig["AppInsightsReactNativePlugin"]: { disabeDeviceCollection: true; }` which is where you will see it after initialization (and also where you can dynamically change it's value with the current version -- as we now expose all of the configurable values by exposing the internal defaults in their "true" location.

The reason it "works" (or at least it should be) being defined in the root (rather than in the extensionConfig) is historical and as long as the extension config is not also defined (by you) we still "support" reading and config specific value from the root of the config object.